stl/time/time/src/main.cpp

```diff
@@ -0,0 +1,8 @@
+#include <iostream>
+#include <string>
+int main()
+{
+    std::string welcome = "Hello World!";
```

Comment:
Indent 2 spaces.

Reply:
Hi, I have changed the code format as requested, can I still get LGTM now?

Reply:
> Indent 2 spaces.
Hi, I have changed the code format as requested, can I still get LGTM now?




> Indent 2 spaces.

Reply:
> Indent 2 spaces.

Hi, I have changed the code format as requested, can I still get LGTM now?